app/models/package.rb

```diff
@@ -721,8 +731,8 @@ def self.bulk_update_attributes(packages,package_attributes)
       results = packages.map do |p|
         p.update_attributes(package_attributes)
       end
-      successes = results.map {|b| b == false }
-      failures = results.map {|b| b == true }
+      successes = results.select {|b| b }
+      failures = results.select {|b| not b }
```

Comment:
Output of previous code...
```ruby
# results -> [false, false, true]
successes = results.map { |b| b == false } # -> [true, true, false]
failures = results.map { |b| == true } # -> [false, false, true]
```
The length of these arrays are returned... and they were the same length... :face_palm: